src/styles/blocks/header.scss

```diff
@@ -0,0 +1,19 @@
+.header {
+  height: $header-hight;
+}
+
+.wrapper {
+  padding: 0 50px;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.logo {
```

Comment:
Create a new file for each BEM block, or if they are related to the header they should be such as `.header__wrapper`